tests/core/icnns_test.py

```diff
@@ -0,0 +1,81 @@
+# coding=utf-8
```

Comment:
To be coherent, I think this test should be in core/nn_potentials

If possible, I would also rename nn_potentials to nn, since this could also be used to define Monge maps (not necessarily a potential).

Reply:
I moved it into `core` and removed the subfolder `nn_potentials`.